Makefile

```diff
@@ -148,7 +148,7 @@ fmt-check: ## Lists formatting issues
 
 .PHONY: imports-check
 imports-check: ## Lists imports issues
-	@test -z $(shell goimports -l $(shell go list -f {{.Dir}} ./... | grep -v proto))
+	@test -z $(shell goimports -l $(shell find . -type f -name '*.go' -not -path './vendor/*' -not -path './cli/cdk/go/proto/*'))
```

Comment:
not platform agnostic but it's fine, I don't think we run this test in CI on windows...

Reply:
Please take a look at the commit message on e043b66030a59f5ad1052afbd55977e914a087c2

It doesn't seem like go-list has a better way to solve this, but I'm open to ideas!